redis/src/cluster_async/mod.rs

```diff
@@ -1196,6 +1182,15 @@ where
         res
     }
 
+    pub(crate) async fn check_topology_and_refresh_if_diff(inner: Arc<InnerCore<C>>) -> bool {
```

Comment:
The return value isn't clear, please add documentation that explains what true/false means

Reply:
made the code self explentory